.circleci/config.yml

```diff
@@ -129,7 +129,7 @@ jobs:
           exclusive: true   # default - true
           # Temporary workaround for Circle CI issue
           # https://discuss.circleci.com/t/setup-remote-docker-connection-failures/26434
```

Comment:
If it'll work, I think we could remove these lines to avoid confusion in the future